src/bin/nydus-image/core/node.rs

```diff
@@ -836,15 +814,15 @@ impl Node {
         Ok(())
     }
 
-    // Xattr paris are located into bootstrap rather than blob, however, we should
-    // also reflect the size they consume like other file system. We don't
-    // directly use local file's metadata for `i_block` since we are purchasing
-    // "reproducible build" which means nydus image can be built from anywhere with
-    // the unique image built.
-    // TODO: The real size occupied within blob is compressed. Therefore, the
-    // sum of all chunks' size should be more accurate. But we don't know the size
-    // right now since compression is not acted yet. Try to make this accurate later.
-    pub fn set_v5_inode_blocks(&mut self) {
+    /// Calculate and set `i_blocks` for inode.
+    ///
+    /// In order to support repeatable build, we can't reuse `i_blocks` from source filesystems,
+    /// so let's calculate it by ourself for stable `i_block`.
+    ///
+    /// Normal filesystem includes the space occupied by Xattr into the directory size,
+    /// let's follow the normal behavior.
+    pub fn set_inode_blocks(&mut self) {
```

Comment:
Maybe still keep the name `set_v5_inode_blocks`? (It's not a blocker for this PR).

Reply:
I have add v5/v6 prefix for v5/v6 dedicated methods:)